conf/cassandra.yaml

```diff
@@ -1467,3 +1467,13 @@ enable_drop_compact_storage: false
 # has exceeded this threshold, returning a read error to the user.
 #client_large_read_warn_threshold_kb: 0
 #client_large_read_abort_threshold_kb: 0
+
+# When client_track_warnings_enabled: true, this tracks the size of the local read (as defined by
+# heap size), and will warn/abort based off these thresholds; 0 disables these checks.
+#local_read_too_large_warning_threshold_kb: 0
+#local_read_too_large_abort_threshold_kb: 0
+
+# When client_track_warnings_enabled: true, this tracks the expected memory size of the RowIndexEntry
+# and will warn/abort based off these thresholds; 0 disables these checks.
+#row_index_size_warning_threshold_kb: 0
+#row_index_size_abort_threshold_kb: 0
```

Comment:
I'm wondering if we should take this opportunity to move all of this stuff into the `SAFETY THRESHOLDS` section above (including `client_track_warnings_enabled`)...

Reply:
it is for safety, but its also `EXPERIMENTAL` given its new... 🤷 

I was thinking about refactoring to make it nested

```
track_warnings:
  enabled: true
  row_index_size:
    warning_threshold_kb: 0
    abort_threshold_kb: 0
...
```

Reply:
we never really got clarity on which way we want to move the project in...

Reply:
I'm 100% in favor of using nesting for new sets of properties when the alternative would be ugly and redundant prefixing. (Honestly surprised I didn't already suggest that here.) I think the conversation in CASSANDRA-15234 was heading toward this as well, but it also has to deal with compatibility, a concern we're more or less free of here.

Reply:
We might even be able to move some of the validation logic into the new config object...

Reply:
Final nice thing about nesting is that we could avoid overloading the thresholds with 0 as the "disabled" value.